mods/ascent/machines/apc.dm

```diff
@@ -0,0 +1,61 @@
+/obj/item/stock_parts/circuitboard/apc/ascent
+	name = T_BOARD("ascent area power controller")
+	build_path = /obj/machinery/power/apc/hyper/ascent/buildable
+
+/obj/item/frame/apc/ascent
+	name = "\improper ascent APC frame"
```

Comment:
Why lowercase `ascent` in these strings?

Reply:
Isn't culture improper when used as a noun?
You don't say British people, you say british people. Maybe I'm misremembering.